### PR TITLE
documentation changes remove pullsecrets and add license key

### PR DESCRIPTION
**Pull request by pns-nirmata**

### CODE REVIEW

File: charts/nirmata/README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ kubectl -n kyverno create configmap <e.g. ca-store-cm> --from-file=custom-ca.pem
 **Install the Kyverno chart:**
 
 ```console
-helm install kyverno --namespace kyverno --create-namespace nirmata/kyverno
+helm install kyverno --namespace kyverno --create-namespace nirmata/kyverno --set licenseManager.licenseKey=<license key obtained from Nirmata support or SAAS>,licenseManager.apiKey=<api key obtained from Nirmata support or SAAS>
```
Comment (atulatnirmata) [inline charts/nirmata/README.md:84]: The prerequisite section `Get token from Nirmata support` should be replaced with a section to get the license key and apikey from support@nirmata.com

File: openshift/README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ $ kubectl create namespace kyverno
 **Install the Kyverno chart:**
 
 ```console
-$ helm install kyverno --namespace kyverno kyverno ./kyverno-charts/nirmata
+$ helm install kyverno --namespace kyverno kyverno ./kyverno-charts/nirmata --set licenseManager.licenseKey=<license key obtained from Nirmata support or SAAS>,licenseManager.apiKey=<api key obtained from Nirmata support or SAAS>
```
Comment (atulatnirmata) [inline openshift/README.md:34]: Add a prerequisite section to get the licensekey and apikey from support@nirmata.com

**Review by atulatnirmata — COMMENTED** [review]
Minor comments

**Review by atulatnirmata — APPROVED** [review]

File: charts/nirmata/README.md (hunk 1 above)
Comment (pns-nirmata) [reply charts/nirmata/README.md]: Changed

File: openshift/README.md (hunk 2 above)
Comment (pns-nirmata) [reply openshift/README.md]: Changed

**Pull request merged**